lib/datastore/index.js

```diff
@@ -374,6 +398,8 @@ Datastore.int = function(value) {
   return new entity.Int(value);
 };
 
+Datastore.NO_MORE_RESULTS = 'NO_MORE_RESULTS';
```

Comment:
There are these 4: https://github.com/GoogleCloudPlatform/gcloud-node/issues/1260#issuecomment-217508895

We won't ever return `NOT_FINISHED`, since we always run it. I'll add the others.